public-api/libproject/devstart.rb

```diff
@@ -797,11 +797,6 @@ def circle_build_cdr_indices(cmd_name, args)
     ->(opts, v) { opts.cdr_version = v},
     "CDR version. Required."
   )
-  op.add_option(
-    "--cdr-date [cdr-date]",
-    ->(opts, v) { opts.cdr_date = v},
-    "CDR date is Required. Please use the date from the source CDR. <YYYY-mm-dd>"
-  )
```

Comment:
you also need to remove any reference to 
opts.cdr_date
\"cdr_date\": \"#{op.opts.cdr_date}\"

Reply:
Pushed the changes.